browser-extension/src/addTournesolRecommendations.js

```diff
@@ -42,6 +52,241 @@ const getParentComponent = () => {
   }
 };
 
+/**
+ * The browser API is expected to return the language indentifier following
+ * the RFC 5646.
+ *
+ * See: https://datatracker.ietf.org/doc/html/rfc5646#section-2.1
+ */
+const isNavigatorLang = (lang) => {
+  let expected = lang.toLowerCase();
+  let found = window.navigator.language.toLocaleLowerCase();
+
+  // `expected` can be the shortest ISO 639 code of a language.
+  //  Example: 'fr'.
+  if (found === expected) {
+    return true;
+  }
+
+  // The shortest ISO 639 code can be followed by other "subtags" like the
+  // region, or the variant. Example: 'fr-CA'.
+  if (found.startsWith(expected + '-')) {
+    return true;
+  }
+
+  return false;
+};
+
+const getLocalizedBannerText = () => {
+  if (isNavigatorLang('fr')) {
+    return (
+      'Le projet Tournesol est-il vraiment efficace? Nous étudions' +
+      " actuellement l'impact de notre extension navigateur sur les" +
+      " habitudes d'utilisation de YouTube. Rejoingnez notre étude" +
```

Comment:
the typo is still present in the extension
```suggestion
      " habitudes d'utilisation de YouTube. Rejoignez notre étude" +
```